AppCenter/AppCenterTests/Util/MSDispatchTestUtil.m

```diff
@@ -12,6 +12,7 @@ + (void)awaitAndSuspendDispatchQueue:(dispatch_queue_t)dispatchQueue {
   // Wait for all tasks to complete, then call suspend in the final task.
   dispatch_semaphore_t semaphore = dispatch_semaphore_create(0);
   dispatch_async(dispatchQueue, ^{
+      
```

Comment:
Unnecessary change even though it has a correct format. The formatter will anyway remove this line in the future.
```suggestion
```